src/nerc_rates/models_outages.py

```diff
@@ -0,0 +1,33 @@
+from typing import Annotated
+from datetime import datetime, timezone
+
+import pydantic
+
+from .models import Base
+
+
+class OutageTimeFrames(Base):
+    start: datetime
+    end: datetime
+
+
+class OutageItemDict(Base):
```

Comment:
This is not actually a dicitonary (unlike `RateItemDict`, which is).

```suggestion
class OutageItem(Base):
```